nestkernel/random_generators.h

```diff
@@ -266,10 +273,18 @@ class RandomGenerator final : public BaseRandomGenerator
     return uniform_ulong_dist_( rng_, param );
   }
 
+  inline unsigned long
+  prand( double mu ) override
+  {
+    std::poisson_distribution< unsigned long >::param_type param( mu );
+    return poisson_ulong_dist_( rng_, param );
+  }
+
```

Comment:
See above

Reply:
Addressed in 29f6d751013b363b90ed241b363c7205d37cfb60.